presto-orc/src/main/java/com/facebook/presto/orc/metadata/StripeInformation.java

```diff
@@ -27,7 +27,6 @@
     public StripeInformation(int numberOfRows, long offset, long indexLength, long dataLength, long footerLength)
     {
         checkArgument(numberOfRows > 0, "Stripe must have at least one row");
-        checkArgument(dataLength > 0, "Stripe must have a data section");
```

Comment:
The reason for removing this line was that if a table may now legitimately have an empty data section.  E.g. if a table just has a map column that is flattened, and every map in that column is empty, there will be no data in the data section (there are no values and every map is non-null so there is no present stream).  I added a unit test for this.